circle.yml

```diff
@@ -0,0 +1,19 @@
+general:
+    artifacts:
+      - .build_ano/uno/firmware.elf
+      - .build_ano/uno/firmware.hex
+
+dependencies:
+    pre:
+        - sudo apt-get install arduino arduino-core arduino-mk
+        - sudo rm -rf /usr/share/arduino/libraries/Robot_Control
```

Comment:
I generally prefer those sorts of comments "this is surprising, but \<reason\>" going inline—it means that one doesn't have to trawl through `git blame` to find out why.